app/views/wizards/claims/add_mentor_wizard/_mentor_step.html.erb

```diff
@@ -0,0 +1,38 @@
+<% caption = current_user.support_user? ? "#{t(".caption")} #{contextual_text}" : t(".caption") %>
+<% content_for :page_title, current_step.errors.any? ? t(".page_title_with_error", error: current_step.errors.first.message, contextual_text: caption) : t(".page_title", contextual_text: caption) %>
+
+<div class="govuk-width-container">
+  <%= form_with(model: current_step, url: current_step_path, method: :put) do |f| %>
+    <%= f.govuk_error_summary %>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <span class="govuk-caption-l"><%= caption %></span>
```

Comment:
See comment above about including school name if it's a support user, and the strangeness occurring on the page.